requirements.txt

```diff
@@ -19,9 +19,10 @@ pyparsing==2.2.0
 python-dateutil==2.7.3
 pytz==2018.5
 six==1.11.0
-SQLAlchemy==1.2.10
+SQLAlchemy
 thespian==3.9.2
-Werkzeug==0.14.1
-PyYAML==5.1
+Werkzeug
+PyYAML
+git+git://github.com/HarsheetKakar/sqlalchemy-paginator.git
```

Comment:
Is this a good idea to use a package that was last updated in 2015?

@chrizandr @Mec-iS

Reply:
we don't want non-ecosystem/external dependencies that are not constantly updated and/or on PyPi

Reply:
its a very small mistake in their code, should i copy the whole code in hydrus itself? Its a small library

Reply:
Doesn't matter if it's a small library or not. If there's a mistake in the main library that clearly indicates that it might not be written very well. Also, it hasn't been updated since 2015.